lib/inspector/diagnostics_node.dart

```diff
@@ -345,7 +356,10 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
     if (value == null) {
       return defaultValue;
     }
-    final level = diagnosticLevelNames[value];
+    final level = nameToDiagnosticLevel[value];
+    if (level == null) {
+      print ('Unabled to find level for $value');
```

Comment:
We should fail in some other way than `print` here. Perhaps just default to some diagnostic level?

Reply:
Updated assert as per @jacob314's suggestion.